COPYING

```diff
@@ -0,0 +1,355 @@
+With the exceptions below, Buildroot is distributed under the terms of
```

Comment:
I don't think the Buildroot disclaimers/prefix are necessary here.

Reply:
Whoops, I assumed the COPYING file in buildroot was generic. I'll make sure I didn't replicate this anywhere else.